client/dashboard/src/components/monaco-editor.tsx

```diff
@@ -1,8 +1,22 @@
-import { useEffect, useRef } from "react";
-import Editor, { OnMount } from "@monaco-editor/react";
-import { useMoonshineConfig } from "@speakeasy-api/moonshine";
 import { cn } from "@/lib/utils";
+import Editor, { loader, OnMount } from "@monaco-editor/react";
+import { useMoonshineConfig } from "@speakeasy-api/moonshine";
 import type * as Monaco from "monaco-editor";
+import * as monaco from "monaco-editor";
+import editorWorker from "monaco-editor/esm/vs/editor/editor.worker?worker";
+import jsonWorker from "monaco-editor/esm/vs/language/json/json.worker?worker";
+import { useEffect, useRef } from "react";
+
+self.MonacoEnvironment = {
+  getWorker(_, label) {
+    if (label === "json") {
```

Comment:
you can use monaco-yaml for monaco support and add the worker definition here

Reply:
Ah i was just assuming you wanted full LSP support (as you were setting up the workers) - but sure yeh we can go without